<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>637</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>439</width>
    <height>637</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>439</width>
    <height>637</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Tile Bank Manager</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="tile_edit_qt.qrc" >
    <normaloff>:/newPrefix/nel.png</normaloff>:/newPrefix/nel.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QGroupBox" name="groupBox" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>0</y>
      <width>421</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title" >
     <string>Land</string>
    </property>
    <widget class="QPushButton" name="editLandPushButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>310</x>
       <y>59</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>Edit Land</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteLandPushButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>310</x>
       <y>97</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>Delete Land</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addLandPushButton" >
     <property name="geometry" >
      <rect>
       <x>310</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>Add Land</string>
     </property>
    </widget>
    <widget class="QListWidget" name="landListWidget" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>20</y>
       <width>291</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>150</y>
      <width>421</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title" >
     <string>Tile Sets</string>
    </property>
    <widget class="QPushButton" name="deleteTileSetPushButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>310</x>
       <y>97</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>Delete Tile Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editTileSetPushButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>310</x>
       <y>59</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>Edit Tile Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addTileSetPushButton" >
     <property name="geometry" >
      <rect>
       <x>310</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>Add Tile Set</string>
     </property>
    </widget>
    <widget class="QListWidget" name="tileSetListWidget" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>20</y>
       <width>291</width>
       <height>211</height>
      </rect>
     </property>
     <property name="editTriggers" >
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
     </property>
    </widget>
    <widget class="QPushButton" name="upPushButton" >
     <property name="geometry" >
      <rect>
       <x>310</x>
       <y>150</y>
       <width>24</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset resource="tile_edit_qt.qrc" >
       <normaloff>:/newPrefix/up.png</normaloff>:/newPrefix/up.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="downPushButton" >
     <property name="geometry" >
      <rect>
       <x>310</x>
       <y>180</y>
       <width>24</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset resource="tile_edit_qt.qrc" >
       <normaloff>:/newPrefix/down.png</normaloff>:/newPrefix/down.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QGroupBox" name="selectedTileSetGroupBox" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>240</y>
       <width>401</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title" >
      <string>TileSet Details</string>
     </property>
     <widget class="QCheckBox" name="orientedCheckBox" >
      <property name="geometry" >
       <rect>
        <x>5</x>
        <y>95</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="layoutDirection" >
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text" >
       <string>Oriented</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resetVegetPushButton" >
      <property name="geometry" >
       <rect>
        <x>370</x>
        <y>20</y>
        <width>28</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="icon" >
       <iconset resource="tile_edit_qt.qrc" >
        <normaloff>:/newPrefix/reset.png</normaloff>:/newPrefix/reset.png</iconset>
      </property>
     </widget>
     <widget class="QLineEdit" name="surfaceDataLineEdit" >
      <property name="geometry" >
       <rect>
        <x>90</x>
        <y>60</y>
        <width>100</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label" >
      <property name="geometry" >
       <rect>
        <x>5</x>
        <y>55</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text" >
       <string>Surface Data:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="chooseVegetPushButton" >
      <property name="geometry" >
       <rect>
        <x>89</x>
        <y>20</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text" >
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3" >
      <property name="geometry" >
       <rect>
        <x>5</x>
        <y>20</y>
        <width>79</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text" >
       <string>Vegetation Set:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>530</y>
      <width>421</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title" >
     <string>Bank</string>
    </property>
    <widget class="QPushButton" name="absolutePathPushButton" >
     <property name="geometry" >
      <rect>
       <x>93</x>
       <y>60</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>60</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize" >
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="text" >
      <string>Absolute Path:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quitPushButton" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>20</y>
       <width>76</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>Quit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exportPushButton" >
     <property name="geometry" >
      <rect>
       <x>335</x>
       <y>20</y>
       <width>76</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>Export...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadPushButton" >
     <property name="geometry" >
      <rect>
       <x>91</x>
       <y>20</y>
       <width>76</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>Load...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="savePushButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>172</x>
       <y>20</y>
       <width>77</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveAsPushButton" >
     <property name="geometry" >
      <rect>
       <x>254</x>
       <y>20</y>
       <width>76</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>Save As...</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="tile_edit_qt.qrc" />
 </resources>
 <connections/>
</ui>
